### PR TITLE
[15 min fix] Update Publishing from RSS guide with correct instructions

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description

This updates https://dev.to/p/publishing_from_rss_guide 

- it adds correct instructions on how to specify the `<category>` XML element, see https://github.com/forem/forem/issues/13753
- updates the layout to be as accessible as possible
- changes "Article" to "Post"
- mentions we also support Atom

## Related Tickets & Documents

Closes https://github.com/forem/forem/issues/13753

### CODE REVIEW

File: app/views/users/_publishing_from_rss.html.erb [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <p>Formatting will typically look good, but you may have to make manual fixes. In the case of Medium, you may have to manually fix embeds.</p>
     <p>
       To find out more about how drafts are created from your feed, read this
-      <a href="/p/publishing_from_rss_guide">guide</a>.
+      <a href="<%= publishing_from_rss_guide_path %>">guide</a>.
```
Comment (mstruve) [inline app/views/users/_publishing_from_rss.html.erb:23]: 💯

**Review by mstruve — APPROVED** [review]
🚀

File: app/views/pages/publishing_from_rss_guide.html.erb [hunk 2]
```diff
@@ -1,12 +1,46 @@
...
+  <link rel="canonical" href="<%= app_url(publishing_from_rss_guide_path) %>" />
+  <meta name="description" content="<%= community_name %> | publishing from rss atom guideline">
   <%= meta_keywords_default %>
 <% end %>
 
-<header>
-  <div class="blank-space"></div>
-</header>
-<%= render "publishing_from_rss_guide_text" %>
+<main class="container article" id="main-content">
```
Comment (aitchiss) [inline app/views/pages/publishing_from_rss_guide.html.erb:9]: 🎉

File: app/views/pages/publishing_from_rss_guide.html.erb [hunk 3]
```diff
@@ -1,12 +1,46 @@
-<% title "Editor Guide" %>
+<% title "Editor Guide - Publishing from RSS or Atom" %>
 
 <%= content_for :page_meta do %>
-  <link rel="canonical" href="<%= app_url("/p/publishing_from_rss_guide") %>" />
-  <meta name="description" content="dev.to | publishing from rss guideline">
+  <link rel="canonical" href="<%= app_url(publishing_from_rss_guide_path) %>" />
+  <meta name="description" content="<%= community_name %> | publishing from rss atom guideline">
```
Comment (aitchiss) [inline app/views/pages/publishing_from_rss_guide.html.erb:5]:
```suggestion
  <meta name="description" content="<%= community_name %> | Publishing from RSS or Atom guidelines">
```

**Review by aitchiss — APPROVED** [review]
Just one wee suggestion - looks great and yay for adding the ID for the skip link 🙌

**Pull request merged**